test/e2e/tests/notifications/account-syncing/importing-private-key-account.spec.ts

```diff
@@ -46,25 +49,24 @@ describe('Account syncing @no-mmi', function () {
             NOTIFICATIONS_TEAM_PASSWORD,
           );
 
-          await driver.clickElement('[data-testid="account-menu-icon"]');
+          const header = new HeaderNavbar(driver);
+          await header.check_pageIsLoaded();
+          await header.openAccountMenu();
 
-          await driver.wait(async () => {
-            const internalAccounts = await driver.findElements(
-              '.multichain-account-list-item',
-            );
-            return internalAccounts.length === accountsSyncMockResponse.length;
-          }, 20000);
-
-          await driver.clickElement(
-            '[data-testid="multichain-account-menu-popover-action-button"]',
+          const accountListPage = new AccountListPage(driver);
+          await accountListPage.check_pageIsLoaded();
+          await accountListPage.check_numberOfAvailableAccounts(
+            accountsSyncMockResponse.length,
           );
-          await driver.clickElement(
-            '[data-testid="multichain-account-menu-popover-add-account"]',
+          await accountListPage.check_accountDisplayedInAccountList(
+            'My First Synced Account',
```

Comment:
I'd love to ditch these magic strings and use something like

```ts
import { SDK } from '@metamask/profile-sync-controller';

const decryptedAccountNames = await Promise.all(
      accountsSyncMockResponse.map(async (response) => {
        const decryptedAccountName = await SDK.Encryption.decryptString(
          response.Data,
          NOTIFICATIONS_TEAM_STORAGE_KEY,
        );
        return JSON.parse(decryptedAccountName).n;
      }),
    );
    
    ...
    
    for (const accountName of decryptedAccountNames) {
      await accountListPage.check_accountDisplayedInAccountList(
        accountName,
      );
    }

```

Reply:
This would apply to all other occurences of magic strings in this PR. Wdyt? 

I just did this for mobile and it works great. Something to be wary of, we'll be adding some decryption workload to our CI by doing that. I'm just not sure how much. Let's discuss!